directus-solidstart/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Getting Started with Directus and SolidStart"
+description: "Let's learn how to build a blog using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items"
+author:
+  name: "Atila Fassina"
+  avatar_file_name: "author.webp"
+---
+
+## Summary
+
+[SolidStart](https://start.solidjs.com) a meta-framework built around [SolidJS](https://solidjs.com) - SolidStart is a framework aiming to allow people and teams to built high-end applications while bringing their own opinions. In this tutorial, you will learn how to build a website using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
+
+## Before You Start
+
+You will need:
+
+- To install Node.js and a code editor on your computer.
+- A Directus project - follow our quickstart guide if you don't already have one.
+- SolidJS knowledge will definitely help, but will not prevent you from following this guide.
+
+## Initialize SolidStart Project
+
+Open your terminal and run the following commands to create a new Solid project and add the Directus JavaScript SDK:
+
+```sh
+npm create solid@latest solid-directus
+```
+
+A prompt will start, here are the sequence of answers provided for this exercise:
+
+```text
+✔ Which template do you want to use?
+  › bare
+
+✔ Server Side Rendering?
+  › yes
+
+✔ Use TypeScript?
+  › no
+```
```

Comment:
````suggestion
```sh
npm create solid@latest solid-directus
✔ Which template do you want to use?
  › bare
✔ Server Side Rendering?
  › yes
✔ Use TypeScript?
  › no
```
````

Combining small steps to shorten the read. Can we also pin this to a version to limit the chance this guide becomes incorrect in the future?